pyproject.toml

```diff
@@ -55,6 +55,7 @@ PyYAML = "^6.0"
 defusedxml = "^0.7.1"
 colorama = "^0.4.6"
 packaging = ">=23.0"
+requests = "2.31.0"
```

Comment:
This version has vulnerability present. Can we instead use Python's stdlib - urllib itself? That way, we won't have this vulnerability present.